### ISSUE TITLE
Convert darknet yolo v4 model to Yolo v5?

### ISSUE THREAD

**Issue by robotaiguy** [ultralytics/yolov5#1303]
## ❔Question
Is there an application that can convert my already existing darknet yolo v4 models to yolo v5 without having to retrain? 

The reason is that the darknet framework seems to have certain cfg parameters (ie: angle, hue, saturation) that augment during training. I don't see these options in yolo v5. I would have to create augmentations in my dataset BEFORE training. Is that correct?

## Additional context
Basically, is there a way to convert .weights to .pt?

**Comment by glenn-jocher** [comment]
@robotwhispering training hyperparameters are located in https://github.com/ultralytics/yolov5/blob/master/data/hyp.scratch.yaml

The ones you referenced are here:
https://github.com/ultralytics/yolov5/blob/bd5cfff8fbdcc079ff901cb7c99fab916be06f57/data/hyp.scratch.yaml#L22-L24

YOLOv5 is not backward compatible with the older darknet cfg's. We use new model yaml files, i.e.:
https://github.com/ultralytics/yolov5/blob/master/models/yolov5s.yaml

**Comment by glenn-jocher** [comment]
Recommend you view https://docs.ultralytics.com/yolov5/tutorials/train_custom_data to get started.
